data/notices.json

```diff
@@ -555,6 +555,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(lambda-nodejs): breaking change in esbuild causes import module error",
+      "issueNumber": 30717,
+      "overview": "Esbuild introduced a breaking change. This caused the import module error in lamda-nodejs module.",
```

Comment:
can we be more specific than the import module error? Users will see this directly, we should include more information

Reply:
Please see the other entries in this file for examples of more extensive descriptions of the issue. The `overview` should be a full summary of the issue; it should describe the problem itself, the root cause, and the impact, including the version numbers it is present on. Think of it as the executive summary of the issue.